sdk/python/tests/test_client.py

```diff
@@ -76,7 +56,6 @@
 
 CORE_URL = "core.feast.example.com"
```

Comment:
do we need this CORE_URL?

Reply:
Yes, it's required to instantiate the Client.